Makefile

```diff
@@ -28,3 +33,7 @@ db:
 		-e MONGO_INITDB_ROOT_USERNAME=${DB_USER} \
 		-e MONGO_INITDB_ROOT_PASSWORD=${DB_PASSWORD} \
 		mongo:latest
+
+# Runs mongo on shell
+db-cli:
+	docker exec -i -t ${DB_CONTAINER_NAME} bash
```

Comment:
Almost, instead of just starting bash and making the user start the mongo CLI just make it so Docker runs the mongo CLI for them.

Make it run this exact command:

```
docker run -it --rm --net host mongo:latest mongo -u ${DB_USER} -p ${DB_PASSWORD}
```

Reply:
command wont work

Reply:
ok works